template/src/routes/profile/index.js

```diff
@@ -1,47 +1,30 @@
-import { h, Component } from 'preact';
-import style from './style';
-
-export default class Profile extends Component {
-	state = {
-		time: Date.now(),
-		count: 10
-	};
-
-	// update the current time
-	updateTime = () => {
-		this.setState({ time: Date.now() });
-	};
-
-	increment = () => {
-		this.setState({ count: this.state.count+1 });
-	};
-
-	// gets called when this route is navigated to
-	componentDidMount() {
-		// start a timer for the clock:
-		this.timer = setInterval(this.updateTime, 1000);
-	}
-
-	// gets called just before navigating away from the route
-	componentWillUnmount() {
-		clearInterval(this.timer);
-	}
-
-	// Note: `user` comes from the URL, courtesy of our router
-	render({ user }, { time, count }) {
-		return (
-			<div class={style.profile}>
-				<h1>Profile: {user}</h1>
-				<p>This is the user profile for a user named { user }.</p>
-
-				<div>Current time: {new Date(time).toLocaleString()}</div>
-
-				<p>
-					<button onClick={this.increment}>Click Me</button>
-					{' '}
-					Clicked {count} times.
-				</p>
-			</div>
-		);
-	}
+import { h } from 'preact';
+import {useEffect, useState} from "preact/hooks";
```

Comment:
We might not want to begin with anything beyond whats in core.

Reply:
Is there any particular reason for this, maybe a long-term maintenance sort of thing? Fair enough of course whatever the reason.

Reply:
I imagine it was the additional weight that we didn't wanted the base template to have. Hooks is an optional feature which is not free.

I'd also like to revisit this decision with @ForsakenHarmony @reznord nad @marvinhagemeister

Reply:
I think hooks is something most people want, so probably not a problem